promql/functions.go

```diff
@@ -1010,6 +1097,12 @@ var functions = map[string]*Function{
 		ReturnType: model.ValVector,
 		Call:       funcMinOverTime,
 	},
+	"month_of_year": {
```

Comment:
Agreed. These are the absolutely sane default cases. If we were to add others, we can then extend by suffixes.